health_vara.py

```diff
@@ -96,13 +117,20 @@ class MammographyPatient(metaclass=PoolMeta):
         'mobile': {'required': False}
     }
 
+    partner_patient_id = fields.Char('Patient ID', select=True)
```

Comment:
part 2. this adds a simple char field for the patient_id, I didn't bother using the party identifiers as the types did not really align to the idea of this field. It seems close to the partner_internal_ids in VaraMG, so have named it partner_patient_id in code.